waspc/data/Generator/templates/react-app/src/entities/index.ts

```diff
@@ -9,6 +9,9 @@ export type {
   {=# entities =}
   {= name =},
   {=/ entities =}
+  {=# isAuthEnabled =}
+  type {= authEntityName =},
```

Comment:
Adding the `Auth` entity to the list of all other entities (easier access for users)